sql/postgres/inspect.go

```diff
@@ -1281,7 +1281,21 @@ SELECT
 	      		ns2.nspname AS referenced_schema_name,
 	      		generate_series(1,array_length(con.conkey,1)) as ord,
 	      		unnest(con.conkey) AS conkey,
-	      		unnest(con.confkey) AS confkey
+	      		unnest(con.confkey) AS confkey,
+				CASE con.confupdtype
+					WHEN 'c' THEN 'CASCADE'
+					WHEN 'n' THEN 'SET NULL'
+					WHEN 'd' THEN 'SET DEFAULT'
+					WHEN 'r' THEN 'RESTRICT'
+					WHEN 'a' THEN 'NO ACTION'
+				END AS update_rule,
+				CASE con.confdeltype
+					WHEN 'c' THEN 'CASCADE'
+					WHEN 'n' THEN 'SET NULL'
+					WHEN 'd' THEN 'SET DEFAULT'
+					WHEN 'r' THEN 'RESTRICT'
+ 					WHEN 'a' THEN 'NO ACTION'
+ 				END AS delete_rule
```

Comment:
I prefer to append the `con.confupdtype`,  `con.confdeltype` to the query and do this mapping in Go.

There are small test cases to change then in `inspect_test.go` and maybe add another one in `internal/integration/testdata/postgres`.

Reply:
Thanks for the feedback @a8m! I had a look at doing it this way, but the problem is that both MySQL and Postgres go through the same code path in `sqlx.SchemaFKs`, and I couldn't find a way to resolve this without more refactoring than I'm comfortable with.

I think handing it over to you is better at this point.